### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by tta2yta**
We created a branch called "feature" for the development of The Next Web clone. 
On the initial commit, we created a basic structure for the page and added CDN links Font awesomely, set up linter, configured it and I also added Google Fonts.
The "Mobile first" approach was used first to achieve responsiveness principles.
Elements were positioned using the grid, flex, and positioning techniques.
The issues raised from the first review of the project in the dashboard are fully addressed.

This project was approved last week but due to some challenges that occurred with the professional skills task, The student success team informed us to repeat it this week.  Could you please review it soon so that we can proceed to the week 3 project as we have already started it by money and we did half part of it?

Please review our project before we merge my branch.
Thank you in advance!

### CODE REVIEW

**Comment by tta2yta** [comment]
we have fixed the linters properly in our local computer but in GitHub, it is rasing an exception of Error: Cannot find module 'postcss/lib/input'.

**Review by MuriloRoque — CHANGES_REQUESTED** [review]
## Status: Changes requested :red_circle: 

### Hello @tta2yta and @prolajumokeoni :wave: 

### You have done an amazing job on this project, it's really good, congratulations :clap: 

### There are only some issues we need to address before moving on, please check my suggestions below:

- [ ] The images on your main 3 articles are not showing, the console informs that they are not being found on your project. Are you sure that the path to them is right on your HTML? Or maybe it's another issue? 

**Screenshot:**

Screenshot from 2021-01-27 13-50-35

- [ ] Add some margin between your sections, they are looking too close to each other, especially on big screens. Check the image below:

Screenshot from 2021-01-27 13-51-07

- [ ] Your linters check is failing, please fix the issues.

**To Highlight :bulb:**

- Your project is well structured;
- Your page is very detailed, almost close to the original;
- Your code is clean, which is a very nice quality.

Kindly make the requested changes and submit them for another review, please :pray: 

I will be on Slack 24/7 in case you have any questions about this review :handshake: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Comment by tta2yta** [comment]
Hello.  Thank you for your comments. We have updated the points you mentioned. Could you please check it again. Thank you for your time.

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,1254 @@
...
+
+  .funding-right ul {
+    display: block;
+  }
+
+  .funding-box .top1 {
+    max-width: 100%;
+    text-align: center;
+
+    /* padding-right: 400px; */
```
Comment (umairarshadbutt) [inline css/style.css]: Remove the commented CSS

File: index.html [hunk 2]
```diff
@@ -0,0 +1,732 @@
...
+            <figure class="fig">
+                    <img alt="The Next Web" src="https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/tnw.svg">
+                    <ul>
+                        <li><a href="">News</a> </li>
+                        <li><a href="">Events</a> </li>
+                        <li><a href="">Index</a> </li>
+                        <li><a href="">Spaces</a> </li>
+                        <li><a href="">Deals</a> </li>
+                        <li><a href="">Answers</a> </li>
+                        <li><a href="">TNW Program</a> </li>
```
Comment (umairarshadbutt) [inline index.html]: Add `#` at empty `href` attributes

File: index.html [hunk 3]
```diff
@@ -0,0 +1,732 @@
...
+                    </div>
+                </div>
+        </div>
+
+        <div class="funding-flex2 flex">
+            <div class="funding-gridd">
+                <div class="funding-box">
+                   <a href="#"><img class="funding-image" src="./img/3.1.png" alt=""></a> 
+                    <p class="top1"><a href="#">BharatPe</a></p>
+                    <h3 class="price"><a href="">₹2B</a></h3>
```
Comment (umairarshadbutt) [inline index.html]: Add `#` at empty `href` attributes

File: index.html [hunk 4]
```diff
@@ -0,0 +1,732 @@
...
+            </div>
+
+        </div> 
+
+    
+        <div class="flex-containar-1 flex">
+            <div class="main1-grid-cont">
+                 <div class="box1 grid-class">
+                     <a class="" href="https://thenextweb.com/neural/2021/01/19/us-regulator-approves-first-ever-fully-autonomous-commercial-drone-flights/">
+                        <img class="logo-inside-grid" src="https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/tnw.svg" alt="">
```
Comment (umairarshadbutt) [inline index.html]: Add the description in the `alt` attribute.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,732 @@
...
+
+
+         <div class="header-flex">
+            <h2 class="main1-header">Latest news ></h2>
+         </div>
+
+        <div class="main-flex flex">
+            <div class="main2-grid-cont">
+                 <div class="grid-boxes">
+                     <a href="#"><img src="./img/1.1.jpg" alt="" class="image-box"></a>
```
Comment (umairarshadbutt) [inline index.html]: Add the description in the `alt` attribute.

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @prolajumokeoni 
Such a remarkable project you made :+1: ! Awesome README :100: !
We, however, do have a few changes to carry out before we proceed (see review comment).

##

 For this project, we have to make sure nothing is messed up. Kindly compare the screenshots below 👇 and work on the suggested changes.
- [ ] Fix the linter issues.
- [ ] Remove the commented CSS.
- [ ] Add `#` at empty `href` attributes. (`href` attribute is empty at 28 times in HTML document)
- [ ] Add the description in the `alt` attribute.(`alt` attribute is empty at 28 times in HTML document)
##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you are stuck at any point or have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Comment by tta2yta** [comment]
Hello. Thank you for your comments. We have updated the points you mentioned. Could you please check it again. Thank you for your time.

we have fixed the linters properly in our local computer but in GitHub, it is rasing an exception of Error: Cannot find module 'postcss/lib/input'.

File: README.md [hunk 6]
```diff
@@ -1,2 +1,55 @@
...
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- <a href="https://www.microverse.org/" target="_blank">microverse.org</a>
+- <a href="https://www.behance.net/gallery/24796463/ZATTIX" target="_blank">ZATTIX Mohammed Awad</a>
+- <a href="https://www.w3schools.com/" target="_blank">w3schools.com</a>
+- <a href="https://www.theodinproject.com/" target="_blank">The Odin Project</a>
+- <a href="https://www.stackoverflow.com/" target="_blank">Stackoverflow</a>
+- <a href="https://css-tricks.com/" target="_blank">CSS-TRICKS</a>
```
Comment (descholar-ceo) [inline README.md]: I would advise you to use markdown language for readme file, not HTML

**Comment by descholar-ceo** [comment]
# This Pull Request is under review by [descholar-ceo](https://github.com/descholar-ceo) :eyes: :eyes: 
## Please wait a bit ...

File: README.md [hunk 7]
```diff
@@ -1,2 +1,55 @@
...
+<a href="https://rawcdn.githack.com/prolajumokeoni/The-Next-Web/9d56f4ded445fa451a1ce60a064939def16ca662/index.html">Live version</a>
+
+## Author
+
+👤 **Oni Olajumoke Priscilla and Tedros Tesfay**
+
+- Github: [Tedros Tesfay](https://github.com/tta2yta)
+- Emaii: tta2yta@gmail.com
+
+👤
```
Comment (descholar-ceo) [inline README.md]: Add a name of the second author

File: README.md [hunk 8]
```diff
@@ -1,2 +1,55 @@
...
+## Live Demo
+
+<a href="https://rawcdn.githack.com/prolajumokeoni/The-Next-Web/9d56f4ded445fa451a1ce60a064939def16ca662/index.html">Live version</a>
+
+## Author
+
+👤 **Oni Olajumoke Priscilla and Tedros Tesfay**
+
+- Github: [Tedros Tesfay](https://github.com/tta2yta)
+- Emaii: tta2yta@gmail.com
```
Comment (descholar-ceo) [inline README.md]: Is it `Emaii`? rather than being `Email` ?

File: README.md [hunk 9]
```diff
@@ -1,2 +1,55 @@
-# The-Next-Web
-A clone application of the next web original website
+
+# HTML/CSS capstone project
```
Comment (descholar-ceo) [inline README.md]: Is this a capstone? Your capstone is yet to come!

File: README.md [hunk 10]
```diff
@@ -1,2 +1,55 @@
...
+
+This project is a responsive website application developed as a completion of HTML & CSS section of Microverse main curriculum. It is a collobtative project intended to showcase HTML & CSS Skills. It is a real-world-like project built with some business specifications. It is a clone of the thenextweb website. It consist of only one page, that is home page. It is adaptive to different standard screen size ( mobile , tablet and desktop). It is developed with HTML and CSS solo.
```
Comment (descholar-ceo) [inline README.md]: This text contains some grammatical and typing errors, please fix them by inspecting the text with [Grammarly](https://app.grammarly.com/), you will be able to spot those errors

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Changes Required :vertical_traffic_light: :warning: 
## Hello @tta2yta and @prolajumokeoni  :handshake: :handshake: :handshake: 

:+1: This is good to make it this far, your clone looks like the original page and it is really nice, congratulations on the good work you have done so far.
:+1: I liked that you have tried to implemented the requested changes from the previous review, keep it up, and your project is really almost done.
#### :warning: However we need to make few changes to mark this project approved, don't feel discouraged because of being asked some changes more and more, that's beacuse this is a learning process, and we are here to help you to become an out-standing developer. See the requested change below :point_down: :point_down: :point_down: 
### Note: All of requested changes which are marked as [Required] are mandatory to be implemented in order to get this project approved, others which are marked as [Optional] are nice to have in your project to make it outstanding

# [Required Changes] :red_circle: 
- [ ] Fix the typos errors from the readme file, you can use [Grammarly](https://app.grammarly.com/) to spot them
- [ ] TWN logo disappears from the top three articles (the leftmost big article) on medium screens 

| Your clone | Original page |
| --- | --- |
| Screenshot from 2021-01-28 10-10-55|Screenshot from 2021-01-28 10-11-55 |

:point_right: Could you see the difference :question: :question: :question: 
___
_If anything seems to be unclear to you, reachout and ask anything by commenting on this Pull Request and mention me as [descholar-ceo](https://github.com/descholar-ceo), I will receive a notification and will respond._

[GitHub](https://github.com/descholar-ceo) | [LinkedIn](https://linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

_Reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)_

**Comment by tta2yta** [comment]
Hello. Thank you for your comment. We have fixed all the points you mentioned regarding the readme file. Regarding the "TWN logo disappears " you said it was not disappearing before in the original website. But we did it as you said. Thank you for your time.

File: README.md (hunk 6 above)
Comment (prolajumokeoni) [reply README.md]: We have implemented this, thank you!

File: README.md (hunk 8 above)
Comment (prolajumokeoni) [reply README.md]: We added links to our Linkedin accounts and removed the emails.

File: README.md (hunk 10 above)
Comment (prolajumokeoni) [reply README.md]: We made some edits to the whole readme file, we hope to hear from you soon, Thank you

**Review by elmejdki — APPROVED** [review]
# status: approved :heavy_check_mark: 

Great job @team, you made a cool clone,

now you can merge your project is approved :checkered_flag: 

## Optional Changes:
but before merging can you please fix this optional requirement below:
:point_down: 

- [ ] Please fix your media queries cause in tablet resolutions it's showing up as if it's on a phone:
image

## Happy Coding :tophat: 
## Zakariae El Mejdki

File: README.md (hunk 10 above)
Comment (descholar-ceo) [reply README.md]: Well done @prolajumokeoni , keep up the good work :muscle: :muscle: :muscle:

**Pull request merged**